utils/ImagesCalculator.js

```diff
@@ -24,13 +24,13 @@ module.exports = async function(action_type, ctx){
 
   function __release(octokit, ctx){
 
-    return ctx.triggered_event.tag_name
+    return Promise.resolve(ctx.event_payload.release.tag_name)
```

Comment:
@frmadem why a Promise.resolve is needed here?

Reply:
Because it is an async block.